packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/ImportNFTs/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useDispatch } from 'react-redux'
+import { colors } from '@blockchain-com/constellation'
+import styled from 'styled-components'
+
+import {
+  Button,
+  Image,
+  Link,
+  SkeletonCircle,
+  SkeletonRectangle,
+  Text
+} from 'blockchain-info-components'
+import CryptoAddress from 'components/CryptoAddress/CryptoAddress'
+import { Flex } from 'components/Flex'
+import FlyoutHeader from 'components/Flyout/Header'
+import QRCodeWrapper from 'components/QRCode/Wrapper'
+import { actions } from 'data'
+import { media } from 'services/styles'
+
+import { Props as OwnProps } from '..'
+
+const ImportNFTs: React.FC<Props> = (props) => {
+  const { close, defaultEthAddr } = props
+  const dispatch = useDispatch()
+
+  const CustomText = styled(Text)`
+    text-align: center;
+    width: 418px;
+    ${media.tablet`
+  width: 100%;
+  `}
+  `
+  const QRCodeContainer = styled.div`
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    width: 100%;
+    padding: 30px 0 0;
+  `
+  const NumberCircle = styled.div`
+    border-radius: 50%;
+    width: 18px;
+    height: 18px;
+    padding: 4px;
+    background: ${colors.grey000};
+    text-align: center;
+  `
+
+  const Address = styled(Flex)`
+    width: fit-content;
+  `
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(defaultEthAddr)
+    dispatch(actions.alerts.displaySuccess('Copied to clipboard!'))
+  }
+
+  return (
+    <>
```

Comment:
Need this? Since you only have 1 child Flex component I think you can just delete this empty fragment component.